packages/syft/proto/core/tensor/tensor.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package syft.core.tensor;
+// import "proto/core/common/common_object.proto";
+// import "proto/core/adp/entity.proto";
+import "proto/lib/numpy/array.proto";
+
+message Tensor {
+  // syft.core.common.UID id = 1;
+  string obj_type = 2;
+  //optional syft.core.adp.Entity entity = 3;
```

Comment:
I'm surprised this works with missing number values. I thought proto didn't like that. Any reason we got rid of entity and id?

Reply:
(this might not have been your doing?)

Reply:
Hmm...the work here is from the branch you had with DP and a lot of the work from here is your work -- I think this one I commented because I only wanted to leave the SMPC part in and then maybe get the DP

Reply:
Is this resolved?